doc/howto/dev/write_docs_cn.rst

```diff
@@ -2,14 +2,13 @@
 如何贡献/修改文档
 ##################
 
-PaddlePaddle的文档包括英文文档 ``doc`` 和中文文档 ``doc_cn`` 两个部分。文档都是通过 `cmake`_ 驱动 `sphinx`_ 编译生成，生成后的文档分别存储在编译目录的 ``doc`` 和 ``doc_cn`` 两个子目录下。
+PaddlePaddle的文档文件都在 :code:`doc` 这个子目录里。源文件是 `RST <http://docutils.sourceforge.net/docs/user/rst/quickref.html>`_ 格式的。 在编译PaddlePaddle源码的时候，可以选择让 `cmake<https://cmake.org/>`_ 调用 `sphinx <http://www.sphinx-doc.org/en/1.4.8/>`_ 从 RST 文件生成 HTML 格式的文档。
```

Comment:
@typhoonzero You are right. I am removing the claim of fixing #2041 from this PR. Also, I am abandoning my change to the Chinese document so to make this PR focusing on "Using Ubuntu 16".